### PR TITLE
Add exclusions of id lang

### PR DESCRIPTION
**Pull request by Mammo32**

### CODE REVIEW

**Comment by Mammo32** [comment]
Add exlusions file

**Comment by Mammo32** [comment]
#1438

**Comment by Mammo32** [comment]
#1438

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "laravel-lang/lang",
+    "name": "mammo32_laravel/lang",
```
Comment (andrey-helldar) [inline composer.json:2]: Why did you change the application namespace?

File: README.md [hunk 2]
```diff
@@ -15,15 +15,9 @@ Install the Laravel-lang package to your Laravel project with composer using com
 
 | Laravel version                     | Composer command                                  |
 |-------------------------------------|---------------------------------------------------|
-| Laravel 8.x                         | `composer require laravel-lang/lang:~7.0` |
-| Laravel 7.x and new directory names | `composer require laravel-lang/lang:~7.0` |
-| Laravel 7.x                         | `composer require laravel-lang/lang:~6.1` |
-| Laravel 6.x                         | `composer require laravel-lang/lang:~5.0` |
```
Comment (andrey-helldar) [inline README.md:21]: Why did you remove the list of supported Laravel versions?

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (andrey-helldar) [reply composer.json:2]: ping

File: README.md (hunk 2 above)
Comment (andrey-helldar) [reply README.md:21]: ping

**Pull request closed without merging**